addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -1013,3 +1013,37 @@ section, .oe_img_bg, [data-oe-shape-data] {
         }
     }
 }
+
+// GRID
+.o_grid_mode {
+    @include media-breakpoint-up(md) {
+        display: grid !important;
```

Comment:
Most rules which are in web_editor.common.scss should actually be in web_editor.frontend.scss (as explained on top of this file). We'll have to took the time to review that at some point but meanwhile: let's not add more mistakes (except if there actually is a reason?) -> move to web_editor.frontend.scss